### DISCUSSION TITLE
Colab Norse won't pip install?

### DISCUSSION THREAD

**Discussion by jmstadt** [norse/norse#388] [Q&A] [answered]
Is anyone having trouble with pip install norse in Colab?  I can install no problem locally in a Jupyter Notebook but when I go to Colab execution starts but never ends.  I verified I can pip install other packages with no problem.

**Comment by jmstadt** [comment, marked as answer]
Never mind on the above, it took a long time but did install

**Comment by jmstadt** [comment]
Of course, now I am getting a module_not_found error for norse_op

**Reply by Jegp** [reply]
Ah, right that can happen when the C++ module isn't installed properly. I'm considering releasing a version without it, actually. Mostly because the `torch.compile` feature seems to compete in terms of performance.
Any help here would be amazing. Either, we can find a way to ignore it in colab environments(?) or we can directly drop it from Norse. But if we do, I'd like to at least run a few tests to make sure the C++ code can be outperformed by `torch.compile`.

**Comment by Jegp** [comment]
We removed the C++ code in favor of `torch.compile`, so the new v1.1.0 should install much faster
